srcpkgs/sisl/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'sisl'
+pkgname=sisl
+version=4.6.0
+revision=1
+wrksrc="SISL-SISL-${version}"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
```

Comment:
this not needed, it's already set by the cmake build style

Reply:
I'll check, but the INSTALL of that lib calls for `ccmake` where you select a release type, I found that a bit odd anyway.

Reply:
You are right, it's on by default, I removed it.